### ISSUE TITLE
missing api-ms-win-downlevel-kernel32-l2-1-0.dll and api-ms-win-core-sysinfo-l1-2-0.dll error

### ISSUE THREAD

**Issue by yarrmateys** [cheat-engine/cheat-engine#2487]
i can't start the 7.5 version. on start it complains about the file api-ms-win-downlevel-kernel32-l2-1-0.dll missing 3 times (and if i go into Windows/System32/CompatTel, grab it and paste it in the ce folder, it then complains about api-ms-win-core-sysinfo-l1-2-0.dll missing, which i don't have), then shows a box to try starting it anyway, but nothing happens.

**Comment by cheat-engine** [comment]
delete the  win32 subfolder from the install dir and see if that helps

**Comment by yarrmateys** [comment]
yep, that helped. it started now.

**Comment by gunxsword** [comment]
主要问题出在DBGHELP.dll这个文件,替换成7.4里面的就解决了,应该也不会有啥问题

**Comment by gunxsword** [comment]
直接删掉的话,那里面还有符号表支持和SQLLITE数据库支持的DLL文件,应该是会对功能有些影响

**Comment by y0umu** [comment]
> delete the win32 and win64 subfolder from the install dir and see if that helps

It works but a violent solution?

**Comment by cheat-engine** [comment]
It will the fallback on the dbghelp dll present in windows. Often a bit outdated but often good enough

**Comment by Dredd893** [comment]
Hello,
I'm having this erorr, but at the same time I don't have win32 and win64 subfolder in the install dir .

**Comment by yarrmateys** [comment]
yeah, removing just dbghelp.dll from both folders is probably better. after removing both winXX folders i wasn't able to scan for pointers, i was getting access violation errors.

**Comment by yarrmateys** [comment]
after some more testing with dbghelp.dll removed from both win32 and 64, it seems to be getting the pointer scanning wrong. after i scanned for pointers, then rescanned for value when they changed, instead of removing all invalid entries and only leaving the valid ones, it seemed to do the opposite, or at least it removed things randomly.

**Comment by Theo1996** [comment]
Same here , windows 7 ultimate, did it drop support for win 7?
